Example/Shared/Config.swift

```diff
@@ -1,5 +1,8 @@
 public enum Config {
-    public static let squareApplicationID: String? = nil // Replace with your squareApplicationID
-    public static let accessToken: String? = nil // Replace with your accessToken
-    public static let locationID: String? = nil // Replace with your locationID
+     static let squareApplicationID: String = "your_square_application_id"
```

Comment:
nit: I kind of liked have these optional because if you ran the app for the first time you would get a compile error saying that you need to input these instead of the app trying to run with dummy values.

Reply:
Good point! Updated back to nil